app/controllers/api/v1/sessions_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class SessionsController < ApplicationController
+      before_action :authenticate
+
+      # TODO: samuel - Bypass CSRF token for now
+      protect_from_forgery with: :exception
+
+      def create
+        user = User.find_by(email: params[:user][:email])
```

Comment:
We can leverage the Rails parameters [API](https://api.rubyonrails.org/classes/ActionController/Parameters.html).

Reply:
Agreed